maquetado/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>HTML & CSS Training</title>
+        <meta name="description" content="training">
+        <meta name="author" content="Wolox">
+        <link rel="stylesheet"  type="text/css" href="application.css">
+    </head>
+    <body>
+        <h1 class="title">Hello World</h1> 
+    </body>
+</html>
```

Comment:
Remember to add an empty new line at the end of text files. [Here's](https://robots.thoughtbot.com/no-newline-at-end-of-file) a nice explanation of why. 😺